GameServerLib/GameObjects/AttackableUnits/AI/ObjAIBase.cs

```diff
@@ -676,5 +676,46 @@ public void SetDashingState(bool state)
         {
             IsDashing = state;
         }
+
+        public int GetObjHash()
+        {
+            var hash = 0;
+            var gobj = "[Character]";
+
+            for (var i = 0; i < gobj.Length; i++)
+            {
+                hash = char.ToLower(gobj[i]) + 0x1003F * hash;
+            }
```

Comment:
These for loops are repetitive blocks of code that implement basic hashes. I'm sure that this "hashing" is already present in LS. Please use the already existing hash function (if it doesn't exist, create one).

Reply:
If I create one, should I use these for loops?